cypress/integration/collections_spec.js

```diff
@@ -138,6 +138,38 @@ describe('Dashboard Collections Page', () => {
       cy.get('#collection-chooser').find(':selected').contains(collectionId);
     });
 
+    it('should copy a collection', () => {
+      const name = 'MOD09GQ';
+      const version = '006';
+
+      cy.visit(`/collections/collection/${name}/${version}`);
+      cy.contains('a', 'Copy').as('copyCollection');
+      cy.get('@copyCollection')
+        .should('have.attr', 'href')
+        .and('include', '/collections/add');
+      cy.get('@copyCollection').click();
+
+      cy.contains('.heading--large', 'Add a collection');
+
+      cy.wait(500);
```

Comment:
Can we eliminate this hard-coded wait time? What is happening that is requiring a wait here? Are we waiting on the collection to be fetched? If so, can we replace this with `cy.wait('@getCollection')`?

Reply:
This one was tricky because it wasn't waiting for a request, but a state update. he `defaultValue` prop changes once we get the schema, and it wasn't seeing the change without the `wait` Since `.contains()` will retry until it finds the element in the DOM, I was able to work around this by instead checking that `.ace__variable` containing the value "name" existed in the DOM.

Reply:
Perfect!